src/fixed-footer-layout.tsx

```diff
@@ -54,10 +55,13 @@ const FixedFooterLayout: React.FC<Props> = ({
     const {isTabletOrSmaller} = useScreenSize();
     const {platformOverrides} = useTheme();
     const {height: realFooterHeight, ref} = useElementDimensions();
-    const windowHeight = useWindowHeight();
+    const isWithinIFrame = useIsWithinIFrame();
+    const windowHeight = useWindowHeight(true);
     const screenHeight = useScreenHeight();
 
-    const hasContentEnoughVSpace = windowHeight - realFooterHeight > screenHeight / FOOTER_CANVAS_RATIO;
+    const hasContentEnoughVSpace =
+        windowHeight - realFooterHeight >
+        (isWithinIFrame ? windowHeight : screenHeight) / FOOTER_CANVAS_RATIO;
```

Comment:
why don't we always use window height?

Reply:
it came from the way we decided to tackle the fixed footer layout issued we used to have with the keyboard open: 
<img width="1019" alt="Screenshot 2023-05-06 at 16 43 35" src="https://user-images.githubusercontent.com/61736059/236631136-2d8a3f10-f39d-4529-b2ad-8f7edea2bfcf.png">

and the PR:
https://github.com/Telefonica/mistica-web/pull/557

if I remember well, screen is more a more stable way of calculating what we need

Reply:
If I remember right, the virtual keyboard + fixed footer  was an issue in Android, not in iOS.
![image](https://user-images.githubusercontent.com/1849576/236757187-644f5fe5-458f-44c3-a875-598cdac897ce.png)
According to this message. If the `availHeight` doesn't change when we open/close the virtual keyboard but the `innerHeight` does change, shouldn't we using the `innerHeight` (`useWindowHeight` in this case)?

Reply:
in case we're into an iframe, the `innerHeight` does not change :): it will be the iframe's height itself
![Screenshot_1683555051](https://user-images.githubusercontent.com/61736059/236846733-e5a723fa-daa7-444f-98fe-8380e4e1806c.png)
![Screenshot_1683555065](https://user-images.githubusercontent.com/61736059/236846739-9dddcfd7-4fef-4d6a-ae6f-9d73de2c5441.png)

I see it convenient using it for this edge case